common/static/css/tinymce-studio-content.css

```diff
@@ -37,7 +37,6 @@
     font-size: 1.6em;
     line-height: 1.4em;
     margin-bottom: 1.6em;
-    text-transform: uppercase;
```

Comment:
why does this go away? Is this the "Preview" label on MCE content editors? or is this something else?

Reply:
I removed a few uppercases because we were advised not to use them for a11y. It can be much harder for some folks to read uppercase text.